feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/ConversationViewModel.kt

```diff
@@ -254,6 +256,19 @@ internal open class ConversationViewModel(
     }
         .stateIn(viewModelScope + coroutineContext, SharingStarted.Lazily)
 
+    val profilePictureImageProvider: StateFlow<ProfilePictureImageProvider?> = flatMapLatest(
```

Comment:
I think this does not belong in the view model necessarily, as it only used in the view.